DigitalLearningSolutions.Web/Views/MyAccount/EditDetails.cshtml

```diff
@@ -11,9 +11,19 @@
 
 @if (User.IsDelegateOnlyAccount()) {
   ViewData["Application"] = "Learning Portal";
+}
+
 @section NavMenuItems {
   <partial name="../LearningPortal/Shared/_NavMenuItems"/>
-}}
+}

```

Comment:
I think these nav menu items should be in the "is delegate" check, otherwise, all accounts will have LP navigation bar